src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -900,9 +900,17 @@
     <comment>An option to choose from for the "bell style" setting. When selected, a combination of the other bell styles is used to notify the user.</comment>
   </data>
   <data name="Profile_BellStyleVisual.Content" xml:space="preserve">

```

Comment:
I thought that your changes made this disappear/no longer necessary.

Let's keep the key, because it needs to be localized for 1.8 still. We just don't have to update it. File a bug to remove it after stable moved to 1.9.